src/python/bot/fuzzers/libFuzzer/stats.py

```diff
@@ -144,8 +144,10 @@ def parse_line_for_strategy_prefix(line, strategy_name):
     if not line.startswith(strategy_prefix):
       return
 
+    prefix_type = strategy.LIBFUZZER_STRATEGIES_WITH_PREFIX_VALUE_TYPE[
```

Comment:
it's `suffix_type` I believe, that prefix is always a string like `corpus_subset_` followed by a value we parse here

Reply:
Done